### PR TITLE
Resolve: [Header] Improve API status

### LINKED ISSUE #1734: [Header] Improve API status
# Currently
Currently we show status of Midgard and different explorers in header as follow:
(1) Offline -> not connected
(2) Locked / not imported -> explorers not connected
(3) Unlocked wallet -> explorers are connected

Screenshot from 2021-09-04 10-14-36

# Want

Screenshot from 2021-09-04 10-32-28

- Remove status of all explores to show Midgard and THORChain API only
  - Offline -> not connected (red icon) 
  - Midgard -> check status using `inbound_addresses` endpoint we call on a regular basis (success -> green icon, error -> yellow icon)
  - THORChain API -> check status using `mimir` endpoint we call on a regular basis (success -> green icon, error -> yellow icon)

### PR DESCRIPTION
**Pull request by gromxyz**
Closes #1734

### CODE REVIEW

File: src/renderer/components/header/netstatus/HeaderNetStatus.tsx [hunk 1]
```diff
@@ -23,70 +24,86 @@ type MenuItem = {
...
   const menuItems = useMemo((): MenuItem[] => {
     const notConnectedTxt = intl.formatMessage({ id: 'setting.notconnected' })
     return [
       {
         key: 'midgard',
         headline: 'Midgard API',
-        subheadline: headerNetStatusSubheadline({ url: midgardUrl, onlineStatus, notConnectedTxt }),
-        color: headerNetStatusColor({ url: midgardUrl, onlineStatus })
+        subheadline: headerNetStatusSubheadline({
+          url: 'midgard.thorchain.info',
```
Comment (veado) [inline src/renderer/components/header/netstatus/HeaderNetStatus.tsx]: Dont't hard-code url, which is different for testnet / mainnet

File: src/renderer/components/header/netstatus/HeaderNetStatus.tsx [hunk 2]
```diff
@@ -23,70 +24,86 @@ type MenuItem = {
...
-        color: headerNetStatusColor({ url: ethereumUrl, onlineStatus })
-      },
-      {
-        key: 'bitcoin cash',
-        headline: 'bitcoin cash',
-        subheadline: headerNetStatusSubheadline({ url: bitcoinCashUrl, onlineStatus, notConnectedTxt }),
-        color: headerNetStatusColor({ url: bitcoinCashUrl, onlineStatus })
+        headline: 'Thorchain API',
+        subheadline: headerNetStatusSubheadline({
+          url: 'thornode.thorchain.info',
```
Comment (veado) [inline src/renderer/components/header/netstatus/HeaderNetStatus.tsx]: Sam here: Dont't hard-code url, which is different for testnet / mainnet

**Review by veado — CHANGES_REQUESTED** [review]
Peek 2021-09-17 10-45

Pls open a browser to link to following urls by clicking an item in the list:

- _Text_: {testnet.}midgard.thorchain.info _Link:_ https://{testnet.}midgard.thorchain.info/v2/doc
- _Text:_ [testnet.}thornode.thorchain.info _Link:_ https://[testnet.}thornode.thorchain.info/thorchain/doc/

File: src/renderer/components/header/Header.util.ts [hunk 3]
```diff
@@ -19,20 +16,14 @@ export const headerNetStatusSubheadline = ({
...
-  )
-}
+}) => (onlineStatus === OnlineStatus.ON ? url : notConnectedTxt)
 
-export type HeaderNetStatusColor = 'green' | 'yellow' | 'red'
-export const headerNetStatusColor = ({ url, onlineStatus }: { url: O.Option<string>; onlineStatus: OnlineStatus }) => {
-  if (onlineStatus === OnlineStatus.OFF) return 'red'
-  return O.isSome(url) ? 'green' : 'yellow'
-}
+export type HeaderNetStatusColor = 'green' | 'red'
```
Comment (veado) [inline src/renderer/components/header/Header.util.ts]: > error -> yellow icon

'yellow'  is still needed, see Details of issue description (part "WANT") https://github.com/thorchain/asgardex-electron/issues/1734

Comment (gromxyz) [reply src/renderer/components/header/Header.util.ts]: Yep, that's fixed just forgot to push it

File: src/renderer/services/clients/common.ts [hunk 4]
```diff
@@ -6,7 +6,13 @@ import { ExplorerUrl$, XChainClient$ } from './types'
 
 export const explorerUrl$: (client$: XChainClient$) => ExplorerUrl$ = (client$) =>
   client$.pipe(
-    RxOp.map(O.map((client) => client.getExplorerUrl())),
+    RxOp.map(
+      O.map((client) => {
+        console.log(client.toString())
+        console.log(client.getExplorerUrl())
```
Comment (veado) [inline src/renderer/services/clients/common.ts]: Remove `log` statements ^

File: src/renderer/components/header/Header.util.ts [hunk 5]
```diff
@@ -31,8 +31,9 @@ export const headerNetStatusSubheadline = ({
   )
 }
 
-export type HeaderNetStatusColor = 'green' | 'yellow' | 'red'
-export const headerNetStatusColor = ({ url, onlineStatus }: { url: O.Option<string>; onlineStatus: OnlineStatus }) => {
-  if (onlineStatus === OnlineStatus.OFF) return 'red'
-  return O.isSome(url) ? 'green' : 'yellow'
-}
+export type HeaderNetStatusColor = 'green' | 'yellow'
```
Comment (veado) [inline src/renderer/components/header/Header.util.ts]: What about `red` for `OnlineStatus.OFF`?

**Review by veado — CHANGES_REQUESTED** [review]
In `OFFLINE` mode all icons needs to be `red`
Screenshot from 2021-09-17 14-26-07

You will see the difference by comparing network status of `v0.4.0` with your changes.

**Review by veado — APPROVED** [review]

**Pull request merged**